nanome/util/config.py

```diff
@@ -55,6 +57,7 @@ def create_parser():
     parser.add_argument('--remote-logging', type=str2bool, default=None, dest='remote_logging', help='Toggle whether or not logs should be forwarded to NTS.')
     parser.add_argument('-r', '--auto-reload', action='store_true', default=None, dest='auto_reload', help='Restart plugin automatically if a .py or .json file in current directory changes')
     parser.add_argument('-i', '--ignore', help='To use with auto-reload. All paths matching this pattern will be ignored, use commas to specify several. Supports */?/[seq]/[!seq]', default='')
+    parser.add_argument('-m', '--monitoring-port', type=int, help='If specified, listens on a port for monitoring purposes')
```

Comment:
Can you add a more detailed description of how this works here? Could also be in the file docstring if you need more space.

Also, is opening a new port going to be an issue for enterprise deploys? Wasn't that the primary reason we proxy the logs through NTS?

Reply:
From when I talked over the phone with Sam, this would be mostly for our public plugins, but that's up for discussion. If we want to deploy this for enterprises, I'll need to make a protocol with NTS.

For the description, feel free to change it, I don't exactly know what details you want to add